src/sentry/snuba/metrics/query_builder.py

```diff
@@ -321,7 +321,7 @@ def get_date_range(params: Mapping) -> Tuple[datetime, datetime, int]:
     if ONE_DAY % interval != 0:
         raise InvalidParams("The interval should divide one day without a remainder.")
 
-    start, end = get_date_range_from_params(params)
+    start, end = get_date_range_from_params(params, default_stats_period=timedelta(days=1))
```

Comment:
This is only used by the metrics web api, not the sessions api, right?

Reply:
yes